src/App.vue

```diff
@@ -71,17 +88,40 @@ body {
 }
 
 .scroll-container {
+  width: 400px;
+  position: relative;
   padding: 98px 0 0;
   overflow: scroll;
   scroll-snap-type: y mandatory;
   scroll-padding: 38px 0 0;
+
+  .resize-handle {
+    content: '';
+    width: 6px;
+    height: 80px;
+    cursor: col-resize;
+    position: absolute;
+    right: 0px;
+    top: calc(50% + 38px);
+    z-index: 9;
+    border-radius: 3px;
+    background-color: var(--box-bg);
+    user-select: none;
+    -webkit-user-drag: none;
+  }
 }
 
 .scrollable {
   scroll-snap-align: none;
   scroll-snap-type: center;
   display: table;
   min-height: calc(100% + 98px);
+  min-width: 350px;
```

Comment:
here as well — scrollable is a utility class, so shouldn't include anything unrelated such as `min-width` or `aside`.